src/Shared/@components/Input/index.js

```diff
@@ -0,0 +1 @@
+export * from './Input';
```

Comment:
Yes, and also `@components/index.js`. And `@shared/index.js`. Yes, @shared, not Shared.

Reply:
Yes, sempai!